lib/inferno/repositories/requirements.rb

```diff
@@ -33,6 +33,7 @@ def insert_from_file(path)
 
         result.each do |raw_req|
           requirement = Entities::Requirement.new(raw_req)
+          next if exists?(requirement.id)
```

Comment:
I don't think we want to do this. We want requirements to be defined in a single location, and test kits should be fixed if if they contain duplicate requirements definitions.

Reply:
makes sense